packages/zmarkdown/public/index.html

```diff
@@ -0,0 +1,201 @@
+
+Pour faire un [lien](http://www.zestedesavoir.com "Zeste de Savoir") sur un morceau de texte
+
+Pour nous contacter, cliquez [ici](mailto:contact@monsite.com).
+
+Nom     |   Age
+------|-----
+Fred |   39
+Sam |   38
+Alice  |   35
+Mathilde  | 35
+
+
++-------+----------+------+
+| Table Headings   | Here |
++-------+----------+------+
+| Sub   | Headings | Too  |
++=======+==========+======+
+| cell  | column spanning |
++ spans +----------+------+
+| rows  | normal   | cell |
++-------+----------+------+
+| multi | cells can be    |
+| line  | *formatted*     |
+|       | **paragraphs**  |
+| cells |                 |
+| too   |                 |
++-------+-----------------+
+
+Nom     |   Age
+------|-----
+Fred |   39
+Sam |   38
+Alice  |   35
+Mathilde  | 35
+Table: Tableau des âges
+
+
+------
+
+
+$$a \cdot x^2 + b \cdot x + c = 0 \quad \Longrightarrow \quad x = \frac {-b \pm \sqrt{b^2 - 4ac}}{2a}$$
+
+Si vous voulez écrire votre formule au sein même d'un paragraphe (comme ceci : $a \cdot x^2 + b \cdot x + c = 0$), alors n'utilisez cette fois qu'un seul caractère dollar `$` avant et après votre formule.
+
+
+```
+#!/usr/bin/env python3
+
+print("Hello, World!")
+```
+
+Avec un langage :
+
+```python
+#!/usr/bin/env python3
+
+print("Hello, World!")
+```
+
+Surligner des lignes :
+
+```perl hl_lines="1 4-6"
+use strict;
+use warnings;
+
+print "Comment vous appelez-vous ? ";
+my $nom = <>; # Récupération du nom de l'utilisateur
+chomp $nom;   # Retrait du saut de ligne
+print "Bonjour, $nom !\n";
+```
+
+![Logo Creative Commons](http://mirrors.creativecommons.org/presskit/logos/cc.logo.png)
+
+!(http://www.youtube.com/watch?v=oavMtUWDBTM)
+
+Utilisez ||Ctrl|| + ||C|| pour copier.
+
+Smileys
+:)
+
+:p :euh:
+
+[[information]]
+| Ceci est une balise d'information.
+|
+| Cool, non ?
+
+> Ceci est une citation
+> sur plusieurs lignes
+Source: Citez vos sources !
+
+
+[[neutre | Théorème de Pythagore]]
+| Un triangle ABC est rectangle en a si et seulement si $AB^2 + AC^2 = BC^2$
+
+
+[[secret]]
+| Ceci est un secret
+|
+| Cool, non ?
+
+
+Bienvenue sur ZdS !
+
+*[ZdS]: Zeste de Savoir
+
+Markdown[^mdown] est une syntaxe légère d'écriture de documents. Pandoc[^pandoc] est un convertisseur de documents.
+
+[^mdown]: Plus d'informations sur [Markdown](http://daringfireball.net/projects/markdown/).
+
+[^pandoc]: Plus d'informations sur [Pandoc](http://johnmacfarlane.net/pandoc/).
+</textarea>
+  </div>
+  <div id="left-bottom" class="split split-vertical"><pre class="bottom-style"><code></code></pre></div>
+</div>
+<div id="center" class="split split-horizontal">
+  <div id="center-top" class="split split-vertical">
+    <div class="result render zmarkdown"></div>
+  </div>
+  <div id="center-bottom" class="split split-vertical"><pre class="bottom-style"><code></code></pre></div>
+</div>
+<div id="right" class="split split-horizontal">
+  <pre class="result render latex-result">
+  </pre>
+</div>
+<script src="../dist/zmarkdown.js"></script>
```

Comment:
This will be an issue to deploy to github pages because this file will be at the root of the branch.

I suggest to delete the current root `public` symlink and creating a symlink from here `./dist -> ../dist`. In the travis deploy script I'll then copy the content before pushing.

Reply:
Here would be the travis config:

```yml
git:
  depth: 10

language: node_js

cache:
  directories:
    - node_modules

node_js:
  - '8'
  - '10'

env:
  - DEST=/tmp
  global:
    secure: V3Wlrzc7mpyVxNxeBPMkfyowycurM2OmkTTc3aeUkRjrdltRNfknudhC2OELSpsfMvUTLLSPW/oMfv+jxgVHOydVIsBbN+lB0jKnm/T+ABfUeDw+tpEEvKvKLtD/uxDB3rC83uD9ociE82PqdGYtBOJX4Sug5bdoq8n02DlsGd1dnsr1dIdIIX0LHp4OkfRF6jv0EyXwSHK7Kk6s/miPtuChHrhJQwsjJBu5YPdSEu+f0mzilEO5NUiv4hT1IWjfBravP6uTyWIUcwtfY1rgmjZy3d+MaCRKbLi4hrzrIa/VL/rX9fg8vN6Jgnt1+kJtPB7Y0yH2tcl0tpeRWW2ppBZOUU2rJ5OWA8TKltZZW7zjQgrdrwNns0+8ifZ7AI/Jdfvfj0NjW/SqsxchLQL2uN0FsBkoUV6Ly4cwZ/B5EIFTdi0qnjcYBdHoXE21j81clk0rKdgiof/lGOL4+bA59O7V88AEOmgGshRF1AUbe1SwBQ6l8+w44ZDg7I9OQfNSf/W07NbT4cwSF+xp4nc0Rw0EzHnfHLxdYv0Dj+1GxAQdtJ22wmN5JFKlENsfslC6bHT2ZN2V5U0K2VqCBdh0iNdV/uV9tdy3GCSnO3W2XuBKtDbo+wrLf8rjIubWwupej8JxR+pPJ10KNJN5rhe/pqIRujr5FxUZJG7mll9w9Ws=

before_install:
  - npm install -g pm2

install:
  - npm install
  - npm run bootstrap

before_script:
  - pm2 ping

after_success:
  - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
  - rm ./packages/zmarkdown/public/js
  - cp ./packages/zmarkdown/dist/* ./packages/zmarkdown/public/js/

deploy:
  provider: pages
  skip-cleanup: true
  github-token: "$GITHUB_TOKEN"
  keep-history: false
  target-branch: gh-pages
  local-dir: ./packages/zmarkdown/public
  on:
    branch: master
```

Reply:
If I well understood what was asked, it's good now.